qiskit_nature/operators/second_quantization/fermionic_op.py

```diff
@@ -210,9 +213,18 @@ def __repr__(self) -> str:
         data = self.to_list()
         if len(self) == 1:
             if data[0][1] == 1:
-                return f"FermionicOp('{data[0][0]}', register_length={self.register_length})"
-            return f"FermionicOp({data[0]}, register_length={self.register_length})"
-        return f"FermionicOp({data}, register_length={self.register_length})"  # TODO truncate
+                data_str = f"'{data[0][0]}'"
+            data_str = f"'{data[0]}'"
+        data_str = f"{data}"
+
+        # TODO truncate
```

Comment:
If this TODO still pending - is it intended to truncate the data string if the data is large - like numpy does when printing arrays?

Reply:
Yes, that's what I intended.

Reply:
Yes. I've already implemented it for the next PR.